src/ResourceManager/Storage/Commands.Management.Storage/help/Set-AzureRmStorageAccount.md

```diff
@@ -391,7 +391,7 @@ Accept wildcard characters: False
 ```
 
 ### -StorageEncryption
-Whether to set Storage Account Encryption KeySource to Microsoft.Storage or not.
+Whether or not to set the Storage account encryption to use Microsoft-managed keys.
```

Comment:
Might be better as "Indicates whether or not to set the Storage account encryption to use Microsoft-managed keys."